operator/bundle_helpers/README.md

```diff
@@ -2,20 +2,18 @@
 
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
 The dependency source files will be prefetched with Cachi2 and made available to the container image build.
-Because GHA/upstream is running on a different Python version, it uses `requirements-gha.txt` to mange its own dependencies.
-Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-gha.txt`.
+Follow the procedure below after any dependencies change for successful builds in Konflux.
 
 ## Prepare the fully resolved requirements files for Cachi2
 
 ### Prerequisite
 
-Run the steps inside a container of the same image as the operator-bundle builder stage.
+Run the steps inside a container of the same image as the [operator-bundle builder stage](./konflux.bundle.Dockerfile).
 
 ```bash
-cd operator/bundle_helpers # if not already in this directory
-docker run -it -v $(pwd):/src --entrypoint /bin/sh registry.access.redhat.com/ubi9/python-39:latest
+cd $(git rev-parse --show-toplevel)/operator/bundle_helpers
+docker run -it -v $(pwd):/src --entrypoint /bin/bash -w /src registry.access.redhat.com/ubi9/python-39:latest
```

Comment:
This is not what I suggested. My suggestion was that while you're anywhere in the repo, you can run the command and it does not move your location.
```suggestion
docker run -it -v "$(git rev-parse --show-toplevel)/operator/bundle_helpers:/src" --entrypoint /bin/bash -w /src registry.access.redhat.com/ubi9/python-39:latest
```

Reply:
done in https://github.com/stackrox/stackrox/pull/14257/commits/817a312c46b790f483f0e147f223493ec2872f67